packages/buildcop/test/buildcop.ts

```diff
@@ -26,11 +26,67 @@ nock.disableNetConnect();
 
 const fixturesPath = resolve(__dirname, '../../test/fixtures');
 
-function formatPayload(payload: BuildCopPayload) {
-  if (payload.xunitXML) {
-    payload.xunitXML = Buffer.from(payload.xunitXML).toString('base64');
-  }
-  return payload;
+function buildPayload(inputFixture: string, repo: string) {
+  const input = fs.readFileSync(
+    resolve(fixturesPath, 'testdata', inputFixture),
+    'utf8'
+  );
+
+  return {
+    repo: `GoogleCloudPlatform/${repo}`,
+    organization: { login: 'GoogleCloudPlatform' },
+    repository: { name: repo },
+    commit: '123',
+    buildURL: 'http://example.com',
+    xunitXML: Buffer.from(input).toString('base64'),
+  };
+}
+
+// Ignore any warning to make it easier to test. No need to include all of the
+// fields from real issues.
+// tslint:disable-next-line: no-any
+function nockIssues(repo: string, issues: any[] = []) {
```

Comment:
If you aren't particular about the type passed in, use `{}` instead of `any`, and you can drop the linter disable.

Reply:
Done. Thanks!